notebooks/RAG_evaluation.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>Let's use an open source model as a judge. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Sadly all the OS models I evaluated fell short in terms accuracy, and I would not rely on them: Prometheus-13B, Mixtral, JudgeLM-33B. But if you have other options I can try them!</p>

Reply:
Oh well...